tokens.xml

```diff
@@ -10956,6 +10979,8 @@ This creature can't be enchanted.</text>
             <reverse-related>Contested Game Ball</reverse-related>
             <reverse-related count="2">Contract Killing</reverse-related>
             <reverse-related>Corsair Captain</reverse-related>
+            <reverse-related>Covetous Elegy</reverse-related>
```

Comment:
I think this should always be x

Reply:
My thought process was that in a two-player game, you are making 1 or 2 tokens since your opponent saved 1 or 2 creatures. In that instance, you might just click the make token option 1 or 2 times, possibly making the x option more trouble that it's worth. But maybe I am overthinking it and just having an x option for making an always unknown amount of tokens is the better idea. I am fine with that change.